module/zfs/dbuf.c

```diff
@@ -941,7 +970,7 @@ dbuf_init(void)
 
 	wmsum_init(&dbuf_sums.cache_count, 0);
 	wmsum_init(&dbuf_sums.cache_total_evicts, 0);
-	for (i = 0; i < DN_MAX_LEVELS; i++) {
+	for (int i = 0; i < DN_MAX_LEVELS; i++) {
```

Comment:
This is the first time I have seen you use C99. ;)

Reply:
I'm warming up to it.  ;)